src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -596,10 +595,6 @@ class GroupReadsByUmi
                          |otherwise discard reads with UMIs shorter than this length and allow for differing UMI lengths.
                          |""")
     val minUmiLength: Option[Int] = None,
- @arg(flag='x', doc= """
```

Comment:
Line 528 in the docstring (and surrounding bulleted list) also needs to be edited.